manifests/kubo.yml

```diff
@@ -117,3 +117,20 @@ variables:
   type: password
 - name: kubelet-password
   type: password
+- name: kubo_ca
```

Comment:
Right now deafult_ca doesn't actually exist in credhub. It only exists in creds.yml. I guess we could add it to credhub after we deploy bosh but I would rather we don't do any manual credential management. What are your hesitations with using a separate CA?